package/src/renderer/__tests__/setup.tsx

```diff
@@ -1,3 +1,5 @@
+import fs from "fs";
```

Comment:
All the changes in this file are to enable `/core` modules to be test (react-native mock, SkiaApi global variable).